src/app/mbs/module-detail/module-detail.component.html

```diff
@@ -70,6 +70,10 @@ <h2 class="title">Module #{{ module.id }}</h2>
         <td scope="row">Rebuild Strategy</td>
         <td scope="row">{{ module.rebuild_strategy }}</td>
       </tr>
+      <tr>
+        <td scope="row">Scratch</td>
+        <td scope="row">{{ module.scratch }}</td>
```

Comment:
I like the approach you took in the other column:
```typescript
{{ module.scratch ? 'Yes' : 'No' }}
```

Reply:
I was thinking the module detail page should be close to a true "dump" of the MbsModule record, which would show the boolean true/false value (or blank if undefined). However, I'm happy to change it to show the canonicalized value.